### PR TITLE
Update DNS example to include sampling>1 note

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

Issue:

https://issues.redhat.com/browse/OSDOCS-10639
Link to docs preview:

https://76678--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/observing-network-traffic.html#network-observability-dns-tracking_nw-observe-network-traffic

QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/network-observability-dns-tracking.adoc [hunk 1]
```diff
@@ -36,7 +36,7 @@ spec:
       sampling: 1              <2>
 ----
 <1> You can set the `spec.agent.ebpf.features` parameter list to enable DNS tracking of each network flow in the web console.
-<2> You can set `sampling` to a value of `1` for more accurate metrics.
+<2> You can set `sampling` to a value of `1` for more accurate metrics and to capture *DNS latency*. For a `sampling` value greater than 1, you can observe flows with *DNS Response Code* and *DNS Id*, and while unlikely, there can be observed *DNS Latency*.
```
Comment (Amoghrd) [inline modules/network-observability-dns-tracking.adoc:39]: wording feels off to me, better to write
`and unlikely that DNS Latency can be observed`

Comment (skrthomas) [reply modules/network-observability-dns-tracking.adoc]: Yea :/ I was unsure tbh if I should even mention it since its unlikely. Addressed.

**Comment by skrthomas** [comment]
@ peer review squad: this PR is being affected by the prow errors. It is a minor update (non-breaking) to a code callout. It was working fine before I changed some wording in the midst of the prow errors. Kathryn says its OK to go ahead and get this peer reviewed, as my release is 6/10 so I really need to get a peer review in. TY!

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**